controller/src/main/java/io/pravega/controller/store/stream/ScaleMetadata.java

```diff
@@ -12,23 +12,27 @@
 import lombok.Data;
 
 import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
 
 @Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
I think this is redundant with the data annotation.

Reply:
Not if you add NoArgsConstructor